src/sim-host/ui/sim-host.css

```diff
@@ -0,0 +1,507 @@
+    font-size: 14px;
+    margin-left: 0;
+    margin-right: 0;
+    resize: none;
+}
+
+body textarea {
+    padding: 2px 4px;
+    width: calc(100% - 10px);
+    /* Account for padding and border */
+ overflow-y: auto;
+}
+
+body input[type =text],body input[type =number] {
+    width: 100px;
+}
+
+body button:active,body textarea:hover,body textarea:active,body input[type =text]:focus,body input[type =number]:focus,body input[type =text]:hover,body input[type =number]:hover {
+    border: 1px solid #CCCCCC;
+    background: #ffffff;
+    font-weight: normal;
+    color: #212121;
+}
+
+body .cordova-state-default {
+    border: 1px solid #d3d3d3;
+    background: #e6e6e6;
+    font-weight: normal;
+    color: #555;
+}
+
+body input:focus,body textarea:focus,body isindex:focus,body keygen:focus,body select:focus {
+    outline: none;
+}
+
+body cordova-radio {
+    display: flex;
+    align-items: center;
+}
+
+body .cordova-panel-row {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+body cordova-group, body cordova-item-list {
+    display: block;
+}
+
+body .cordova-radio-wrapper.cordova-inline {
+    display: inline-flex;
+}
+
+body .cordova-radio-label {
+    padding-left: 0.25em;
+}
+
+body cordova-item + cordova-item {
+    display: block;
+    border-top: #d3d3d3 1px solid;
+    margin-top: 10px;
+    padding-top: 10px;
+}
+
+body cordova-item:hover:before {
+    content: url(delete.png);
+    position: absolute;
+    margin-left: auto;
+    right: 0;
+    background: white;
+    font-size: 14px;
+    padding-left: 8px;
+    font-weight: bold;
+    color: darkred;
+    cursor: pointer;
+}
+
+body .cordova-collapse-icon {
+    cursor: pointer;
+    background: url('collapse.png') no-repeat 50% 50%;
+    width: 22px;
+    height: 22px;
+    position: absolute;
+    margin-top: 1px;
+    margin-bottom: auto;
+    right: 6px;
+    margin-left: auto;
+}
+
+body .cordova-collapse-icon.cordova-collapsed {
+    background: url('expand.png') no-repeat 50% 50%;
+}
+
+body .cordova-collapse-icon:hover, body .cordova-close-icon:hover {
+    background-color: rgba(255, 255, 255, 0.25);
+}
+
+body .cordova-close-icon {
+    cursor: pointer;
+    background: url('close.png') no-repeat 50% 50%;
+    width: 22px;
+    height: 22px;
+    position: absolute;
+    margin-top: 1px;
+    margin-bottom: auto;
+    right: 6px;
+    margin-left: auto;
+}
+
+body .cordova-content {
+    overflow: hidden;
+    transition: height 0.2s;
+}
+
+/* Styles to enable feedback link in a footer panel */
+.feedbackFooter {
+    background: rgba(255, 255, 255, 0.85);
+    height: 30px;
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    box-shadow: 0 -1px 1px 1px rgba(0, 0, 0, 0.2);
+    z-index: 1000;
+}
+
+.feedbackLink {
+    position: absolute;
+    top: 5px;
+    right: 10px;
+}
+
+.cordova-main {
+    padding-bottom: 40px;
+}
+
+/* Chrome 29+ */
```

Comment:
I agree with @TimBarham on this, duplicating styles will make the maintenance and improving the sim host UI harder than keeping the express middleware. I believe that we can leave with that middleware for a while, knowing that is a hack and that in the future (not so far) will be removed.
If it happens that we need to keep the `deep` combinator for Chrome, then we can think of other approaches.